docs/releases/1.10.txt

```diff
@@ -467,6 +467,18 @@ Templates
 * Added relative path support for string arguments of the :ttag:`extends` and
   :ttag:`include` template tags.
 
+* To keep similar functionality from Django 1.8-9: add a :setting:`TEMPLATES` dict e.g.::
```

Comment:
This doesn't belong in the "Minor features" section. I think the closest related item is in the "Features removed" section where it says, "The following settings are removed". The current advice you've added is a bit simplistic if any TEMPLATE_* settings are customized. I guess linking to `ref/templates/upgrading.txt` would be more appropriate.